Dockerfile

```
@@ -1,16 +1,7 @@
-# Base AWS Java app image
-
-ARG JAVA_HOME=/usr/local/openjdk-11
-FROM openjdk:11-jre-slim AS jre
-
-FROM amazonlinux:2.0.20190508
-WORKDIR /app
+FROM amazonlinux:2.0.20191016.0
 
 # Install Java
-ARG JAVA_HOME
-ENV JAVA_HOME $JAVA_HOME
-ENV PATH=$PATH:$JAVA_HOME/bin
-COPY --from=jre $JAVA_HOME $JAVA_HOME
+RUN yum install -y java-11-amazon-corretto
```

JAVA_HOME needs to be added as an environment variable to make it work.

```suggestion
RUN yum install -y java-11-amazon-corretto
ENV JAVA_HOME /usr/lib/jvm/java-11-amazon-corretto.x86_64
```